ratis-netty/src/main/java/org/apache/ratis/netty/server/DataStreamManagement.java

```diff
@@ -170,10 +170,12 @@ public String toString() {
   static class StreamMap {
     static class Key {
       private final ChannelId channelId;
+      private final ClientId clientId;
       private final long streamId;
 
-      Key(ChannelId channelId, long streamId) {
+      Key(ChannelId channelId, ClientId clientId, long streamId) {
```

Comment:
Could we remove channelId since we have clientId?  If yes, we can use ClientInvocationId and remove the entire StreamMap.Key class.

Reply:
Yes, we can remove channelId